client/my-sites/stats/site.jsx

```diff
@@ -233,26 +251,33 @@ class StatsSite extends Component {
 		if ( isDateControlEnabled ) {
 			// Sort out end date for chart.
 			const chartEnd = this.getValidDateOrNullFromInput( context.query?.chartEnd );
+
 			if ( chartEnd ) {
 				customChartRange = { chartEnd };
 			} else {
 				customChartRange = { chartEnd: moment().format( 'YYYY-MM-DD' ) };
 			}
+
 			// Sort out quantity for chart. Default to 30 days.
-			let daysInRange = 30;
+			let daysInRange = this.getDefaultRangeForPeriod( period );
```

Comment:
Should update or remove the comment to keep it consistent with the code. Probably not needed now that we have a descriptive function name in place.